### PR TITLE
adding updated hyrax spec form matcher to confirm license is multi-va…

### PR DESCRIPTION
**Pull request by jenlindner**
…lued

### CODE REVIEW

File: spec/views/hyrax/base/_form_metadata.html.erb_spec.rb [hunk 1]
```diff
@@ -134,8 +134,8 @@
     end
 
     it 'has licenses' do
-      expect(page).to have_form_field(:license)
-        .as_single_valued
+      expect(page)
+        .to have_multivalued_field(:license)
```
Comment (no-reply) [inline spec/views/hyrax/base/_form_metadata.html.erb_spec.rb:138]: `#have_multivalued_field` is deprecated. Fields are now assumed multi-valued unless `.as_single_valued` is specified.

File: spec/support/matchers/have_form_field.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
```
Comment (no-reply) [inline spec/support/matchers/have_form_field.rb:1]: Can we just release `hyrax-spec` `0.3.1`, rather than copying this in?

**Review by no-reply — CHANGES_REQUESTED** [review]
Can we merge https://github.com/curationexperts/hyrax-spec/pull/13 and release a version instead of copying gem code up locally?

File: spec/support/matchers/have_form_field.rb (hunk 2 above)
Comment (jenlindner) [reply spec/support/matchers/have_form_field.rb:1]: @no-reply sure, i just didn't want to overlook it.

**Pull request closed without merging**